calldwell/rust_helpers.py

```diff
@@ -0,0 +1,162 @@
+"""Module containing some boilerplate, common to all tests that use `calldwell-rs` library."""
+import logging
+from typing import Any, Callable, Optional, Tuple
+
+from .gdb_client import GDBClient
+from .rtt_client import RTTClient
+
+RTT_SECTION_SYMBOL_NAME = "_SEGGER_RTT"
+"""Section name of RTT symbol. Hard-coded in `rtt_target` library."""
```

Comment:
Shouldn't the docs be above the line they are documenting?

Reply:
Not in Python